code/controllers/subsystem/familytree.dm

```diff
@@ -89,6 +94,8 @@ SUBSYSTEM_DEF(familytree)
 		if(FAMILY_FULL)
 			if(H.virginity)
 				return
+			if(H.age == AGE_CHILD)
```

Comment:
AssignToHouse then return also add to newly wed or they still become a wife or husband